data-raw/ictr_data.R

```diff
@@ -3,10 +3,8 @@ library(readr)
 library(usethis)
 
 # The raw data comes from tiltIndicator#87, which includes pre-processing
```

Comment:
@kalashsinghal, is this comment still valid? 
Where did you get this latest version of the example datasets?

Reply:
@maurolepore changed it to this PR tiltIndicator #170 (given by Tilman on slack)